pkg/apis/kubeone/validation/validation.go

```diff
@@ -39,9 +39,9 @@ import (
 
 const (
 	// lowerVersionConstraint defines a semver constraint that validates Kubernetes versions against a lower bound
-	lowerVersionConstraint = ">= 1.27"
+	lowerVersionConstraint = ">= 1.28"
```

Comment:
This is a nice comment.

It doesn't help me at all to understand what the constraint _means_. This constant isn't offering any abstraction in itself.

Maybe `minSupported` is a better name or the comment could explain `// ... defines the lowest Kubernetes version supported by KubeOne.`.